lerna.json

```diff
@@ -3,5 +3,7 @@
   "npmClient": "yarn",
   "changelogPreset": "@okta/conventional-changelog-odyssey",
   "useWorkspaces": true,
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "useNx": false
```

Comment:
This is false, but I see an nx.json. I don't know much about nx, can you give some insight into this?

Reply:
I should remove the nx.json.

Reply:
It looked like this, but it's gone now:
```json
{
  "tasksRunnerOptions": {
    "default": {
      "runner": "nx/tasks-runners/default",
      "options": {
        "cacheableOperations": ["build", "test", "svgo", "start"]
      }
    }
  },
  "targetDefaults": {
    "build": {
      "outputs": ["{projectRoot}/dist"]
    },
    "typecheck": {
      "dependsOn": ["^build"]
    }
  }
}
```

Reply:
82c29667dd9b8b7f8ee27cccafc93c010ac31a38